web/source/keyboards/kmwkeyboards.ts

```diff
@@ -978,18 +1009,27 @@ namespace com.keyman.keyboards {
                 }
               }
             } else { // Single language element
-              tEntry = new CloudRequestEntry(x[i]['id'], x[i]['languages'][j]['id']);
+              tEntry = new CloudRequestEntry(x[i]['id'], x[i]['languages']['id']);
               if(this.isUniqueRequest(cloudList, tEntry)) {
                 cloudList.push(tEntry);
               }
             }
           }
+
+          // TODO: Convert stub from one-to-many KeyboardStub[]
+          // Involves mapping KeyboardStub properties
+          keyboardStubs.push(stub);
```

Comment:
1.  This should not be left in as a TODO, at least not without promise of followup work shortly thereafter to implement what's needed.
2.  That "mapping KeyboardStub properties" is kinda necessary, since `stub` itself (as noted) does not adhere to the `KeyboardStub` type specification.  (See prior comment.)

When both those points combine... I'm fairly certain this should be done _within this PR_.  Shouldn't even add much complexity to do so; it's just self-contained detail work.

Reply:
I've started to address this TODO in https://github.com/keymanapp/keyman/pull/5389/commits/e01f6bf20179462f50cc15edb703ed704772f6d6

@jahorton  Can you tell me what "region code" property maps to `KRC`?